.github/workflows/pr.yml

```diff
@@ -51,15 +53,14 @@ jobs:
           submodules: 'recursive'
       - name: Setup Ninja
         uses: seanmiddleditch/gha-setup-ninja@master
-      - name: Build Realm Dart for Linux
+      - name: Build Realm for Linux
```

Comment:
I'm okay with this change, but the library being build is `librealm_dart.so`. I have wondered, if it would be better to build two libs, ie. also a `librealm.so`, so that the later can be shared across SDKs, with the Dart VM specific parts only, being in `librealm_dart.so`. Anyway - that is not for this PR, just something to think about.

Reply:
This is the name of the library used by both flutter and dart - at least looking at the [latest release](https://github.com/realm/realm-dart/releases/tag/0.2.1%2Balpha), all libraries except for iOS are librealm_dart. I'm fine with renaming it to librealm, but I agree it should be done as a separate PR as it will likely be more involved than what I wanted to do in this one.

Reply:
Why are we thinking renaming the library. I think we are totally sidetracking here. It's fine to rename the step.

Reply:
Not sure I understand - why would we rename the step? This is building the native Realm library for Linux. It's used by both dart and flutter, which is why I think it's incorrect to call it "Build Realm Dart".

Reply:
Also, I don't think it's particularly important to rename the library and in any case, not right now.

Reply:
yes indeed.